app/utils/database.py

```diff
@@ -398,6 +398,8 @@ class Meta:  # pylint: disable=too-few-public-methods
 
         indexes = ((("mother", "father", "birth_date"), True),)
 
+## Create sequence to allow unique ids for digital certificates
+DATABASE.execute_sql('''CREATE SEQUENCE IF NOT EXISTS certificates_seq START WITH 100000 INCREMENT BY 1 MAXVALUE 199999 NO CYCLE''')
```

Comment:
Move this to `init`?

I think sqlite doesn't support `CREATE SEQUENCE`, so only run if using postgres. 

(How does this match with tests?)

Reply:
It should be now in init(). I am not sure what to do about sqlite.